api/src/main/java/org/jenkinsci/plugins/workflow/flow/GraphListener.java

```diff
@@ -27,6 +27,8 @@
 import org.jenkinsci.plugins.workflow.graph.FlowNode;
 
 /**
+ * A low-level implementation of the Flow listener.
+ * A classic Jenkins extension point is implemented as {@link FlowListener}.
```

Comment:
It's problematic due to the current plugin's API. It's hard to prevent usage conflicts since there are explicit methods for adding GraphListeners